service/application/src/main/java/com/solace/maas/ep/runtime/agent/scanManager/ScanManager.java

```diff
@@ -65,7 +70,9 @@ public String scan(ScanRequestBO scanRequestBO) {
                         .stream())
                 .collect(Collectors.toUnmodifiableList());
 
-        return scanService.singleScan(routes, routes.size(), scanId);
+        loggingService.prepareLoggers(groupId, scanId, messagingServiceEntity.getId());
```

Comment:
This hasn't addressed my previous point about not creating the Logger for each scan in specific methods. We need to use the object created by Logback and not create a Logger for each scan initiated.